website/static/css/custom.css

```diff
@@ -61,7 +61,7 @@
     height: 50%;
 }
 
-.doc-button {
+.try-button {
```

Comment:
Maybe we can make the button less wide so we have smaller margins now that the text is shorter.

Reply:
Done

Reply:
I think we can make the font size 18px or 20px.

Reply:
I think the 20px looks better.
Done